docs/articles/other/Account-Management.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Account Management
+description: Everything you need to know about Having an Expensify Account Manager
+---
+
+<!-- The lines above are required by Jekyll to process the .md file -->
+
+# What is an Account Manager?
+An Account Manager is a dedicated point of contact to support customers with questions about their Expensify account. They will actively monitor open technical issues and be proactive with recommendations to increase efficiency and minimize time spent on expense management.
+
+Unlike Concierge, an account manager’s support will not be real-time, 24 hours a day.  A benefit of Concierge is that you get real-time support every day. Your Account Manager will be super responsive when online, but anything sent when they’re offline will not be responded to until they’re online again.
+
+For real-time responses and simple troubleshooting issues, you can always message our general support by writing to Concierge via the in-product chat or by emailing concierge@expensify.com.
+
+# How do I know if I have an account manager?
+If you are a policy or domain admin, you will hear from your account manager as soon as one gets assigned to your company. You’ll also have the option to contact them when you log in to your account and click the Concierge icon.
+
+# How do I contact my account manager?
```

Comment:
Looks good @johncschuster! Just a question, I noticed in the doc we have these three topics for the LHN:
- What is an Account Manager?
- How do I contact my Account Manager?
- FAQs

But when I test locally the section `How do I contact my account manager?` appears in the root level, is that correct? Or should we add this section as a sub-section of `How do I know if I have an account manager?`?

<img width="1789" alt="Screen Shot 2022-10-26 at 11 36 50" src="https://user-images.githubusercontent.com/6829422/198084285-04fe82a6-5768-4177-855d-6c5ad2462977.png">


```suggestion
## How do I contact my account manager?
```

Reply:
Eagle eyes! You're spot on. I'll commit your suggestion.